src/components/PlaceholderContainer/PlaceholderContainerAction.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+
+import type {ButtonSize, ButtonView} from '../Button';
+import {Button} from '../Button';
+import {block} from '../utils/cn';
+
+import {componentClassName} from './constants';
+
+import './PlaceholderContainer.scss';
+
+const b = block(componentClassName);
+
+export interface Action {
+    text: string;
+    loading?: boolean;
+    disabled?: boolean;
+    view?: ButtonView;
+    size?: ButtonSize;
+    handler?: (event: React.MouseEvent<HTMLButtonElement>) => void;
+    href?: string;
+}
+
+interface PlaceholderContainerActionProps {
+    action: Action;
+}
+
+export const PlaceholderContainerAction = ({action}: PlaceholderContainerActionProps) => {
```

Comment:
There is no advantages to keep this markup as separate file, it has no styles and simply proxy props for Button